docs/src/main/docs/microprofile/02_server-configuration.adoc

```diff
@@ -57,23 +58,34 @@ ssl:
     keystore-passphrase: "abcd"
 ----
 
+== Configure "admin" port
+Helidon can expose multiple ports. For the purpose of Helidon MP, this support has the following limitations:
```

Comment:
Using just `Helidon` in the begining is a bit ambiguous, since this is document is for Helidon MP only this can be more explicit.
Also the use of `purpose` is a little awkward. 

How about `Helidon MP can expose multiple ports, with the following limitations:`